tokens/color/status.yaml

```diff
@@ -0,0 +1,45 @@
+color:
+  status:
+    $extensions:
+      com.redhat.ux:
+        order: 500
+
+    danger:
+      on-light:
+        $value: '{color.red-orange.50}'
+        $description: Danger accent color (light theme)
```

Comment:
this produces `--rh-color-status-danger-on-light`, but the changeset mentions `--rh-color-danger-on-light`. which one reflects your intentions, the changeset or the yaml record?